libraries/AP_Mission/AP_Mission_ChangeDetector.cpp

```diff
@@ -43,7 +43,7 @@ bool AP_Mission_ChangeDetector::check_for_mission_change(uint8_t& first_changed_
             mis_change_detect.cmd[num_cmds-1] = cmd[num_cmds-1];
             if (!cmds_changed) {
                 cmds_changed = true;
-                first_changed_cmd_idx = cmd_idx;
```

Comment:
Not a big deal but this part of this commit should really be in, "AP_Mission_ChangeDetector: returns index of first command changed".  This doesn't change the final result though.